### ISSUE TITLE
IIS Web App Manage Task - WebDAV module being enforced even when "disabled"

### ISSUE THREAD

**Issue by JCanlett** [Microsoft/vsts-tasks#7838]
VSTS IIS Web App Manage Task

## Issue Description
HTTP error code 405 on all PUT & DELETE request post release. 

Per the docs - "The task updates websites, webapps and application pools but does not install or configure IIS web server on the machines."

However the first option is "Enable IIS" which appears runs the cmndlet Add-WindowsFeature web-server -includeallsubfeature cmd which I'm not familiar with.

After release completes using this check box, even if IIS was enabled on the server all PUT AND DELETE requests trigger a 405 result. For every app on the IIS server, not just the targeted site. Also note that WebDAV Module was set to "disabled" for the affected site.

**Comment by rajatagrawal-dev** [comment]
@JCanlett If you enable "Enable IIS" option, then all the sub-features of "Web Server (IIS)" feature will get enabled, including WebDAV publishing. So please disable WebDAV manually and re-deploy you website with "Enable IIS" option **disabled**. See if that works.

**Comment by JCanlett** [comment]
@rajatagrawal-dev  Yes that is exactly what we had to do. Just a case of features going faster than documentation I guess. Also note that while the installation is "seemless" the uninstall is not and requires server restart. So this was a bit of an issue for our team.
